apps/redi-connect/src/pages/front/signup/SignUp.tsx

```diff
@@ -61,6 +56,8 @@ export interface SignUpFormValues {
   firstName: string
   lastName: string
   agreesWithCodeOfConduct: boolean
+  mentor_isPartnershipMentor?: boolean
+  mentor_workPlace?: string
```

Comment:
Should we also add these to the initial values? Or we do not add those in case they are optional?

Reply:
I like the idea of doing it - with `mentor_isPartnershipMentor = false`. No need with `mentor_isWorkPlace` since it won't always appear to the user.